scss/bootstrap/components/_checkbox-switch.scss

```diff
@@ -0,0 +1,153 @@
+/* Variables */
+
+$checkbox-switch-speed:       0.2s;
+$checkbox-switch-gap:         2px;
+$checkbox-switch-size-small:  10px;
+$checkbox-switch-size-normal: 20px;
+$checkbox-switch-size-medium: 30px;
+$checkbox-switch-size-large:  40px;
+
+/* Basic (default) styles */
+
+.checkbox-switch {
```

Comment:
I'm curious about why you decided to "extend" `.checkbox`, what's the reasoning behind it?